### ISSUE TITLE
node.js v6.11.2 (LTS) compatibility

### ISSUE THREAD

**Issue by mn-martin** [kriskbx/gitlab-time-tracker#16]
In the readme it says node.js version >= 6 is supported: https://github.com/kriskbx/gitlab-time-tracker/blob/master/readme.md#requirements

That's not true.

Here's the error I get with node.js macOS (x64) v6.11.2 LTS:

```
gtt log
/usr/local/lib/node_modules/gitlab-time-tracker/include/tasks.js:112
        return new Promise(async function (resolve, reject) {
                           ^^^^^

SyntaxError: missing ) after argument list
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/local/lib/node_modules/gitlab-time-tracker/gtt-log.js:9:15)

```

Now that I have v8.4.0 installed, everything`s working. I think you should try to maintain support for node.js LTS.

**Comment by kriskbx** [comment]
Yeah, you're right: supporting 6.11 LTS is actually my goal for gtt. Unfortunately I'm using 8.4 locally so things like this can happen if you don't have a proper testing or continuos integration setup. I'm currently working on a travis ci setup for testing against different node versions and will hopefully integrate unit testing soon: so things like this won't happen anymore. You can expect an update that will fix this within this week.

**Comment by mn-martin** [comment]
That's very good news, thank you @kriskbx

**Comment by ibotty** [comment]
Can you reopen this bug to see when running it with node6.11 is possible?

**Comment by kriskbx** [comment]
I just released version `1.6.5` which is again compatible with Node.js 6.11. I also integrated a first set of unit tests as well as travis ci to prevent something like this from happening again. :)
